analyzers/ruby/ruby_test.go

```diff
@@ -43,3 +45,48 @@ func TestCustomGemfileLockPath(t *testing.T) {
 
 	assert.NotEqual(t, analyzed, analyzedCustom)
 }
+
+func TestFallbackOnMissingBundler(t *testing.T) {
+	buildTarget := "github.com/fossas/fossa-cli/cmd/fossa"
```

Comment:
This is incorrect. As documented in the package comment for the Ruby analyser, the build target of a Ruby module is the directory containing the project. In this case, it should be `testdata`. (Tests are always run with the CWD set to the test file's folder.)